com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/schema.py

```diff
@@ -24,6 +24,7 @@ class CommonSchema(enum.Enum):
     Json = StreamSchema("tuple<rstring jsonString>")
     String = StreamSchema("tuple<rstring string>")
     Binary = StreamSchema("tuple<blob binary>")
+    PythonHash = StreamSchema("tuple<blob __spl_po,int32 __spl_hash>")
```

Comment:
Any suggestions on where to put it?  Should I just create a new class ?
class InternalSchema(enum.Enum):
    """Internal stream schemas"""
    PythonHash = StreamSchema("tuple<blob __spl_po,int32 __spl_hash>")

    def schema(self):
        return self.value.schema();

    def spl_json(self):
        return self.value.spl_json()